text/0008-sync-and-storage-review-packet.md

```diff
@@ -0,0 +1,371 @@
+# Roadmap Review: Sync and Storage
+
+**Firefox should wow users with a smart personalized experience that's available seamlessly across services, apps, and devices. User data storage and syncing are vital to this vision.**
+
+We propose to reframe user profile data as a pillar of the Firefox ecosystem, rather than continuing to allow our storage and syncing story to emerge piecemeal from the implementation details of each new product feature.
+
+User data becomes a deliberately designed service layer on which many different product experiences are built. We foster through design reviews the long-term importance of user data. And we invest technically in a more unified, extensible, flexible, cross-platform suite of storage and sync solutions that support a coherent set of approaches to storage.
+
+## Review question
+
+Is this roadmap proposal congruent with Firefox's strategy?
+
+## Document status
+
+*   2017-11-20: Go-ahead given.
+*   2017-11-09: Review successfully completed. Follow-up questions underway.
+*   2017-11-03: Sufficient contributions received; sent to the Reviewer.
+
+## Roles
+
+<dl>
+
+<dt>Reviewer</dt>
+
+<dd>dcamp</dd>
+
+<dt>Chair</dt>
+
+<dd>jwalker</dd>
+
+<dt>Proposers</dt>
+
+<dd>
+
+*   rnewman, Browser Architecture
+*   rfkelly, Engineering Lead, Firefox Accounts
+*   markh, Sync engineering
+*   emily, Browser Architecture
+*   adavis, Product Manager, Sync & FxA
+
+</dd>
+
+</dl>
+
+# Lay Summary
+
+Firefox, and the other new experiences that we want to create, rely on easily recording, combining, syncing, extending, and repurposing user data.
+
+Desktop Firefox’s data is spread across [45+ different stores using 10+ different storage technologies](https://github.com/mozilla/firefox-data-store-docs), and we keep adding more.
+
+In general, these stores and technologies:
+
+*   Were not designed for syncing, for reuse on other platforms, or for their data to be repurposed.
```

Comment:
Is there a reason for `*<SPACE><SPACE><SPACE>Text` rather than `*<SPACE>Text`? I'm guessing it's an artifact of the way you did the export. But the rest of the repo does the latter.

Reply:
Yeah, just tooling.